tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -5896,6 +5903,8 @@ node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(83,9
 node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(84,7): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(84,34): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(84,63): error TS2555: Expected at least 2 arguments, but got 1.
+node_modules/chrome-devtools-frontend/front_end/emulation/DeviceModeView.js(103,12): error TS2339: Property '_model' does not exist on type 'typeof DeviceModeView'.
```

Comment:
Type lookup is getting static type of class type reference instead of instance type. This is not right.

(The type reference is a superfluous `this` reference, so the program is actually better without it.)